packages/docusaurus-plugin-content-docs/src/docs.ts

```diff
@@ -270,6 +270,21 @@ async function doProcessDocMetadata({
 
   const draft = isDraftForEnvironment({env, frontMatter});
 
+  const formatDate = (locale: string, date: Date, calendar: string): string => {
+    try {
+      return new Intl.DateTimeFormat(locale, {
+        day: 'numeric',
+        month: 'long',
```

Comment:
At any rate, I'd prefer this to be `short`, like `Jun 19, 2020`. MDN and the TypeScript website both use this style.